### ISSUE TITLE
Need to put a wildcard on jmeter-plugins in order to load libs in jmeterPluginJars

### ISSUE THREAD

**Issue by djangofan** [kulya/jmeter-gradle-plugin#43]
The 3rd party library include requires that I used the syntax "jmeter-plugins.*"  .   I think the intention here was to only be required to use "jmeter-plugins", just like the other libraries.    This request is just to adjust the regular expression so that loading jmeter-plugins is more like loading the other libraries.

This is the workaround until it is fixed:
jmeterPluginJars = ["groovy","guava","opencsv","jmeter-plugins.*","commons-lang3"]

Also, you might want to put a snippet of documentation in your README on how to do this, for other people.   It is a very important aspect of using Jmeter.

**Comment by foragerr** [comment]
> just like the other libraries

I just looked at your pom.xml here: https://github.com/djangofan/launch-jmeter/blob/master/pom.xml#L47
you seem to be calling out each jmeter plugin explicitly by name?

**Comment by djangofan** [comment]
I am confused as to why you are referring to the pom.xml  in a project where I am only referring to the build.gradle file.

**Comment by foragerr** [comment]
I apologize, I should've been clearer. I was curious how other build-plugins implemented jmeter-plugin handling, in particular how maven-jmeter plugin does it - so I looked at your pom. 

More relevant to this issue: If you listed each plugin by its full module-name like "jmeter-plugins-common"  in build.gradle, (like you did in pom.xml), it would still work without the need for wildcards or code-change.

This is neither a case for, or against your suggestion; just an observation.

**Comment by foragerr** [comment]
I agree with your stance on documentation, starting a wiki page.
https://github.com/kulya/jmeter-gradle-plugin/wiki

**Comment by djangofan** [comment]
Yes, I see what you mean.  Good point.   I supposed I could do that.  I was trying to account for that part of the kulya code that is matching the jmeter-plugins, where it seems it was intended to be done as a group wildcard.
